src/OpenLoco/TownManager.cpp

```diff
@@ -59,6 +63,61 @@ namespace OpenLoco::TownManager
 
     static auto& rawTowns() { return getGameState().towns; }
 
+    // 0x00497348
+    void sub_497348()
```

Comment:
`resetBuildingsInfluence` or is that too much of a mouthful? This function is only called from Town::initialise and GameCommand::removeTown.